src/core/permissions/sagas/updatePermissions.js

```diff
@@ -32,32 +38,47 @@ function* updatePermissionsWorker(action :SequenceAction) :Saga<WorkerResponse>
   try {
     yield put(updatePermissions.request(action.id, action.value));
 
-    const {
+    const permissionsUpdates :Array<{
+      actionType :ActionType;
+      permissions :Map<List<UUID>, Ace>;
+    }> = action.value;
+
+    const updateAclsCalls = [];
+
+    permissionsUpdates.forEach(({
       actionType,
       permissions,
     } :{
       actionType :ActionType;
       permissions :Map<List<UUID>, Ace>;
-    } = action.value;
+    }) => {
```

Comment:
You don't need to specify the flow type here as it's already defined above, and since it's a simple array, flow knows what types `actionType` and `permissions` are.